src/rnnlm/rnnlm-training.h

```diff
@@ -69,7 +69,7 @@ class RnnlmTrainer {
   */
   RnnlmTrainer(bool train_embedding,
                const RnnlmCoreTrainerOptions &core_config,
-               const RnnlmEmbeddingTrainerOptions &embedding_config,
+               const RnnlmEmbeddingTrainerOptions &/* unused embedding_config*/,
```

Comment:
It seems like it is used, but not via the class member, instead it is given to the trainer like this:
```
    embedding_trainer_ = new RnnlmEmbeddingTrainer(embedding_config,
                                                   embedding_mat_);
```
(which keeps it as a const reference).
I think we can remove the class member.

Reply:
Thanks! Could you please look at another one: https://github.com/kaldi-asr/kaldi/pull/4560/files#diff-b3543f4e3f4dc99d000d8a05e473e533426da611c5eb58e660d6f85fe5347817 , the `TransitionModel OnlineWakeWordFasterDecoder::trans_model_` member in online2/online-nnet3-wake-word-faster-decoder.h? I never looked at the wake word stuff, and am not even sure what the term means exactly. If it what I think it is, an okaygoogle bracketed by silence, then it should be a more or less distant kin to decoder's endpointing, then the transition model would be likely needed.

The rest of suspiciously unused stuff is in @hugovbraun's _UnübertroffenezauberlicheCUDAkunst_ realm.